### PR TITLE
Return labels for FER2013 if possible

### PR DESCRIPTION
**Pull request by NicolasHug**
Closes https://github.com/pytorch/vision/issues/8118
Closes https://github.com/pytorch/vision/pull/8368
Closes https://github.com/pytorch/vision/pull/8415

If either the `fer2013.csv` file or  `icml_face_data.csv` file is present, we use one of them instead of `train.csv` and `test.csv`, and return the labels for the test data instead of None. `fer2013` has precedence over the rest, then `icml_face_data.csv` is used, and then `train/test.csv` are used.

### CODE REVIEW

File: torchvision/datasets/fer2013.py [hunk 1]
```diff
@@ -47,14 +52,25 @@ def __init__(
                 f"https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge"
             )
 
+        pixels_key = " pixels" if use_icml else "pixels"  # yes, for real
```
Comment (pmeier) [inline torchvision/datasets/fer2013.py:55]: Oh FFS. Took me two minutes to notice the extra space. I share your feelings here, but maybe point out the space in the comment?

**Review by pmeier — APPROVED** [review]
LGTM. Since the files cannot be downloaded by us, maybe add a sentence to the docstring on how to get test labels?
